pandas/io/gbq.py

```diff
@@ -128,23 +130,44 @@ def test_google_api_imports(self):
         except ImportError as e:
             raise ImportError("Missing module required for Google BigQuery support: {0}".format(str(e)))
 
-    def get_credentials(self):
+    def authorize(self):
```

Comment:
Ok, I think #11335 may be a better solution since it allows the user to pass in a json key. I'm going to close this PR based on the discussion in #8489